conf/modules/phasing.config

```diff
@@ -26,56 +26,85 @@ process {
 
     withName: '.*:PHASING:HIPHASE_SNV' {
 
-        ext.args = "--ignore-read-groups"
+        ext.args = { "--ignore-read-groups --stats-file ${meta.id}.stats.tsv --blocks-file ${meta.id}.blocks.tsv --summary-file ${meta.id}.summary.tsv" }
```

Comment:
How can I use `[...].join(' ')` on these?

Reply:
```suggestion
        ext.args = { [
        '--ignore-read-groups',
        "--stats-file ${meta.id}.stats.tsv",
        "--blocks-file ${meta.id}.blocks.tsv",
        "--summary-file ${meta.id}.summary.tsv"
        ].join(' ') }
```
Doesn't this work? 
I think we do something similar to this in tomte 
https://github.com/genomic-medicine-sweden/tomte/blob/4047a72316213a490f8b0d738f0450a7ad699b45/conf/modules/alignment.config#L39

Reply:
Yes 🤦, just had the `{...}` wrong